src/main/java/com/project/shop/item/controller/ItemController.java

```diff
@@ -17,20 +17,20 @@
 @RestController
 public class ItemController {
 
-    private final ItemService itemService;
+    private final ItemServiceImpl itemServiceImpl;
```

Comment:
ItemService 를 인터페이스로 추상화하여 Impl에서 구현하는 시도가 좋았네요 :) 
추상화를 한 이유는 다형성을 이용하기 위함이겠죠!? 하지만 코드에는 다형성을 이용하지 않았어요.
아래 내용 고민해보고 같이 이야기해봐요.
- 왜 다형성을 적용했는지?
- 다형성을 적용한 후 장점과 단점이 무엇인지?

```suggestion
    private final ItemService itemServiceImpl;
```

Reply:
다형성을 이용하면 기능의 변화나 리팩토링의 이유로 메소드 수정이 이루어져도
메소드 호출부는 수정 사항 없이 구현부만 수정하면 된다. 
하지만 현재 서비스는 한 개의 클래스로 이루어져 있고, 서비스가 늘어날 가능성이 적기 때문에 itemService에서 구현하고 호출하게 한다.